src/styles/globals.css

```diff
@@ -1,3 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
```

Comment:
Maybe we could use the Next next/font built-in feature?
[next/font](https://beta.nextjs.org/docs/optimizing/fonts)

Also we can combine it with tailwind
What do you think?

Reply:
Ok, thanks, I will add it this way 👍🏻

Reply:
Done